mainapp/templates/mainapp/request_list.html

```diff
@@ -80,9 +80,10 @@ <h2 class="text-center">ഇതു വരെ ആവശ്യപ്പെട്ട
         <th>Request Number - അപേക്ഷ നമ്പര്‍</th>
         <th>District - ജില്ല</th>
         <th>Location - സ്ഥലം</th>
-        <th>Requestee - അപേക്ഷകന്‍റെ പേര</th>
+        <th>Requestee - അപേക്ഷകന്‍റെ പേര്</th>
         <th>Phone - ഫോണ്‍ നമ്പര്‍</th>
         <th>Date - തീയതി</th>
+        <th>Elapsed Time - അപേക്ഷ നൽകിയ സമയം </th>
```

Comment:
The translation is not quite right. അപേക്ഷ നൽകിയ സമയം = Time of application right

Reply:
ok will change it. This was picked from the issue ticket.

Reply:
@vvazhayil let me know what would be the official wordings and i can change.

Reply:
Make it : കഴിഞ്ഞു പോയ സമയം